parent/pom.xml

```diff
@@ -230,16 +233,27 @@
         <geronimo-jta-spec-version>1.1.1</geronimo-jta-spec-version>
         <geronimo-ws-metadata-spec-version>1.1.3</geronimo-ws-metadata-spec-version>
         <gmetric4j-version>1.0.10</gmetric4j-version>
-        <google-guava-version>19.0</google-guava-version>
+        <google-guava-version>28.1-jre</google-guava-version>
```

Comment:
Do NOT change this as other components are dependent on this old version. Instead create a new for pubnub (guava hell)

Reply:
Right, I think I can reuse the grpc-guava-version property which is on the right version